package.json

```diff
@@ -2,34 +2,33 @@
   "name": "TO-FIX",
   "version": "1.0.0",
   "description": "A task manager for OpenStreetMap",
-  "main": "src/index.js",
+  "private": true,
   "dependencies": {
     "d3": "^3.5.5",
-    "leaflet-omnivore": "0.3.2",
-    "leaflet-osm": "^0.1.0",
-    "mapbox.js": "2.1.9",
+    "lodash": "^4.16.4",
+    "mapbox-gl": "^0.26.0",
+    "normalizr": "^2.2.1",
     "osm-auth": "^0.2.8",
-    "random-strings": "0.0.1",
-    "raven-js": "^1.1.17",
-    "react": "^0.13.1",
-    "react-keybinding": "^1.1.1",
-    "react-router": "^0.13.2",
-    "reflux": "^0.2.7",
-    "store": "^1.3.17",
-    "wellknown": "^0.3.1",
-    "xhr": "2.0.1"
+    "react": "^15.3.2",
+    "react-addons-pure-render-mixin": "^15.3.2",
+    "react-addons-update": "^15.3.2",
+    "react-dom": "^15.3.2",
+    "react-keybinding": "^3.0.0",
+    "react-redux": "^4.4.5",
+    "react-router": "^3.0.0-beta.1",
+    "redux": "^3.6.0",
+    "redux-logger": "^2.7.0",
+    "redux-thunk": "^2.1.0",
+    "turf": "^3.0.14",
```

Comment:
Fixed in [7c81712](https://github.com/osmlab/to-fix/pull/160/commits/7c81712285d40c23398eb0fe0d0eed2224868e08).

Replacing turf and lodash with specific submodules decreased the build size by 100KB. 🚀